tests/llm/test_gpt4all.py

```diff
@@ -0,0 +1,64 @@
+import os
+
+import pytest
+from langchain.llms.gpt4all import GPT4All as LangchainGPT4All
+
+from embedchain.config import BaseLlmConfig
+from embedchain.llm.gpt4all import GPT4ALLLlm
+
+
+@pytest.fixture
+def config():
+    os.environ["OPENAI_API_KEY"] = "test_api_key"
```

Comment:
I dont think you need to set this env variable for the open source llm setup.

Reply:
Fixed